src/extension.js

```diff
@@ -23,8 +23,12 @@ function _isDirectory(file) {
 function createBreadCrumbItemsFromFile(fileName, callback) {
     // this wall of code full of shit but do exactly what it should
     // no power to refactor it
+    fileName = path.normalize(fileName)
```

Comment:
Can u please add semicolon at the end of new lines (26, 28, 44)?
It'll make this PR same formatted with rest of code)

Reply:
Sure!